### ISSUE TITLE
Console command color codes are not included in the command

### ISSUE THREAD

**Issue by SurvivalQuest** [SkriptLang/Skript#2464]
### Description
When executing a console command that contains a color/format code, the code is not included in the actual command, but instead the text after it is formatted.

### Steps to Reproduce
command /color [<text>]:
  permission: color.use
  permission message: &4Insufficient permission.
  trigger:
    if arg-1 is not set:
      send "&cUsage: /color [Color]"

    if arg-1 is "dark_blue" or "1":
      execute console command "/pex user %player% prefix ""&1test"""
      message "Your name is now &1%player%"

### Expected Behavior
The console command would include &1 in the command.

### Errors / Screenshots
http://puu.sh/EiRMG/cbc2e90d40.png

### Server Information
* **Server version/platform:** git-Paper-192 (MC: 1.14.4)
* **Skript version:** 2.4-beta7

**Comment by Govindass** [comment]
I'm having a similar problem, there seems to be issues with color codes everywhere where variables are used, for example:
send "&e%{_hello}%hi"
would output a yellow <none> with white hi, variables reset color codes

**Comment by Wealthyturtle** [comment]
Duplicate of #2319

**Comment by SurvivalQuest** [comment]
I don’t believe this is a duplicate of that issue. I’m not having an issue where variables reset color codes. The issue is that the console command is not printing the actual color code characters in the string.

**Comment by Wealthyturtle** [comment]
Sorry, misunderstood the issue. It's not a duplicate, but I'm sure this is not a bug? Arguments will be formatted by default, so that you can do something like
```
command /setmotd <text>:
    trigger:
        set {motd} to arg-1 #Will be colored
```
If you want the argument to be uncolored, but still retain the &1 format code, I don't think such an expression exists at the moment
